sharding-sql-test/src/main/java/org/apache/shardingsphere/test/sql/loader/SQLCasesLoader.java

```diff
@@ -190,7 +193,7 @@ private String getLiteralSQL(final String sql, final List<?> parameters) {
     }
     
     private static Collection<String> getDatabaseTypes(final String databaseTypes) {
-        return Strings.isNullOrEmpty(databaseTypes) ? getALlDatabaseTypes() : Splitter.on(',').trimResults().splitToList(databaseTypes);
+        return Strings.isNullOrEmpty(databaseTypes) ? getALlDatabaseTypes() : Lists.newArrayList(Splitter.on(',').trimResults().split(databaseTypes));
```

Comment:
What is the different with origin code?

Reply:
1. splitToList  retrun readOnly List ,if  use add, remove ,can throws exception.
2. init list No default capacity is specified，so this method is @beta。

Reply:
Why getDatabaseTypes need to modify by user?